### PR TITLE
LES-44-checkbox

### PR DESCRIPTION
**Pull request by OxygenO22**

### CODE REVIEW

File: src/compopnents/checkbox/Checkbox.module.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    margin: 10px;
+}
+
+.checkbox__label {
+    display: flex;
+    align-items: center;
+}
+.checkbox__input {
+    margin-right: 10px;
+}
```
Comment (Evgeniy121212) [inline src/compopnents/checkbox/Checkbox.module.css:17]: good. upgrade to .scss

**Review by Evgeniy121212 — COMMENTED** [review]
good, but please, remove unnecessary files from that PR - it can affects other peoples

leave only checkbox files:\
`src/compopnents/checkbox/Checkbox.tsx` (! implement .tsx)\
`src/compopnents/checkbox/Checkbox.module.scss` (! change to .scss)

(and rename folder `src/compopnents` -> `src/components`)

**Comment by NatashaMikhaltsova** [comment]
please, remove unnecessary files (src/App.js, src/components/checkbox/Checkbox.module.css), classes should be used from Checkbox.module.scss

**Review by Evgeniy121212 — CHANGES_REQUESTED** [review]
please, remove unnecessary files

it must be only checkbox files:\
`src/compopnents/checkbox/Checkbox.tsx` (! implement .tsx)\
`src/compopnents/checkbox/Checkbox.module.scss` (! change to .scss)

(and rename folder `src/compopnents` -> `src/components`)

**Comment by OxygenO22** [comment]
Done, I deleted the unnecessary file

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (Terapool) [inline src/App.js:1]: Please, rename this file to 'App.tsx' as we write our project in Typescript.

**Comment by NatashaMikhaltsova** [comment]
Note that unnecessary files are still here 
image

Also folder with components should be named 'components', not 'components/checkbox'. If subfolder 'checkbox'  is required by @Evgeniy121212, put created 2 files (Checkbox.tsx, Checkbox.module.scss) to it.

File: src/components/checkbox/Checkbox.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React, {useState} from 'react';
+import classes from './Checkbox.module.scss';
+
+const Checkbox = (props) => {
+    const [checkbox, setCheckbox] = useState(false);
```
Comment (Terapool) [inline src/components/checkbox/Checkbox.tsx:5]: const [isChecked, setIsChecked] = ...

it is better to use "is...", "has..." for such props and state naming

**Review by Terapool — CHANGES_REQUESTED** [review]

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (OxygenO22) [reply src/index.js:1]: done

File: src/App.js (hunk 2 above)
Comment (OxygenO22) [reply src/App.js:1]: done

File: src/components/checkbox/Checkbox.module.scss [hunk 5]
```diff
@@ -0,0 +1,17 @@
+* {
```
[the remark this answers is not in the document]
Comment (OxygenO22) [reply src/components/checkbox/Checkbox.module.scss:1]: done

File: src/components/checkbox/Checkbox.module.scss [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-size: 16px;
+}
+.checkbox__wrapper {
```
[the remark this answers is not in the document]
Comment (OxygenO22) [reply src/components/checkbox/Checkbox.module.scss:7]: done

File: src/components/checkbox/Checkbox.tsx (hunk 3 above)
Comment (OxygenO22) [reply src/components/checkbox/Checkbox.tsx:5]: done

File: src/App.js (hunk 2 above)
Comment (OxygenO22) [reply src/App.js]: done

**Review by Terapool — APPROVED** [review]

**Review by NatashaMikhaltsova — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 7]
```diff
@@ -1,24 +1,13 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <form>
+            <Checkbox/>
+      </form>
+
```
[the remark this answers is not in the document]
Comment (OxygenO22) [reply src/App.tsx:10]: done

[the remark this answers is not in the document]
Comment (NatashaMikhaltsova) [reply src/App.tsx:10]: remove commented code

**Review by NatashaMikhaltsova — CHANGES_REQUESTED** [review]

**Review by DimitarShemet — APPROVED** [review]

**Review by Kurilo-Pavel — APPROVED** [review]

**Review by NatashaMikhaltsova — APPROVED** [review]

**Review by valer-sky — APPROVED** [review]

**Comment by Evgeniy121212** [comment]
closing because another PR was created #21

**Pull request closed without merging**